plugin/trino-hive/src/test/java/io/trino/plugin/hive/s3/TestS3WrongRegionPicked.java

```diff
@@ -36,11 +36,9 @@ public void testS3WrongRegionSelection()
                     .setHiveProperties(ImmutableMap.of("hive.s3.region", "eu-central-1")) // Different than the default one
                     .build()) {
                 String tableName = "s3_region_test_" + randomNameSuffix();
-                queryRunner.execute("CREATE TABLE default." + tableName + " (a int) WITH (external_location = 's3://" + bucketName + "/" + tableName + "')");
-                assertThatThrownBy(() -> queryRunner.execute("SELECT * FROM default." + tableName))
+                assertThatThrownBy(() -> queryRunner.execute("CREATE TABLE default." + tableName + " (a int) WITH (external_location = 's3://" + bucketName + "/" + tableName + "')"))
                         .rootCause()
-                        .hasMessageContaining("Status Code: 400")
-                        .hasMessageContaining("Error Code: AuthorizationHeaderMalformed"); // That is how Minio reacts to bad region
+                        .hasMessageContaining("Status Code: 400");
```

Comment:
Can we restore the error code and the message so that code readers can understand the cause?

Reply:
I extended error message.
But now flow is different before we bypass this check, but now failing on `isDirectory` call.